### PR TITLE
Configures package.json for npm consumption

### PR DESCRIPTION
**Pull request by Ollo**
- Adds basic button partial to test integration

cc/ @saralohr @williamdingwall

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "description": "a CSS component library for TeamSnap",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/teamsnap/teamsnap-ui.git"
+  },
+  "author": "",
```
Comment (Ollo) [inline package.json]: Wasn't sure if there was a `design@teamsnap.com` alias or something that would be useful here.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "teamsnap-ui",
+  "version": "0.1.0",
+  "description": "a CSS component library for TeamSnap",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (Ollo) [inline package.json]: This section has some options to write specific action that should take place after pushing to git, installing into a project etc.. <https://docs.npmjs.com/misc/scripts>

File: src/Button/Button.scss [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.Button {
```
Comment (Ollo) [inline src/Button/Button.scss]: Just put this in as a point to test with.. directory structure is all up in the air.

Comment (mikereinmiller) [inline src/Button/Button.scss]: was just about to comment on that.  Good call, I'm good with this for initial testing.  Nice work.

**Pull request merged**